client/src/js/components/bhDebtorGroupSelect.js

```diff
@@ -40,15 +40,18 @@ function DebtorGroupSelectController(DebtorGroup) {
 
     // load all Debtor Group
     DebtorGroup.read(null, filters)
-      .then(function (debtorGroups) {
-        $ctrl.debtorGroups = debtorGroups;
-      });
+      .then(handleDebtorGroups);
 
     $ctrl.valid = true;
   };
 
+  function handleDebtorGroups(debtorGroups) {
```

Comment:
Could you make this configurable?  We use debtor groups selects in a couple places (reports, search modals) where they aren't necessarily required.  The original issue proposes a flag on the component, kind of like `excludeTitleAccounts` on `bhAccountSelect`.  What do you think?

Reply:
We can put the message at the bottom : 
![image](https://user-images.githubusercontent.com/5445251/33938611-64b64d5c-e008-11e7-8b3b-2bd3631d3d48.png)

The alert message can be shown only if the attribute `warn-no-group` is true. The input box is always displayed